src/utils/transform.js

```diff
@@ -346,7 +362,7 @@ export const extractPropsFromState = (store, component, customOptions) => {
 		categoryValue: store.internalValues[component]
 			? store.internalValues[component].category
 			: undefined,
-		value,
+		value: value || undefined,
```

Comment:
@mohdashraf010897 It should be specific to `suggestion` type of queries. We use value as empty string for `search` type of queries which would break with this change.